### PR TITLE
Fix the problem that can't get return value from echo %OS% cmd in pktgen test case

### PR DESCRIPTION
**Pull request by Zhengtong**
ID: 1337028

### CODE REVIEW

File: qemu/tests/pktgen.py [hunk 1]
```diff
@@ -113,7 +113,8 @@ def run(test, params, env):
     utils_misc.verify_host_dmesg()
 
     error.context("Ping external host after pktgen test", logging.info)
-    status, output = utils_test.ping(dest=external_host, session=session,
+    session_ping = vm.wait_for_login(timeout=login_timeout)
```
Comment (Zhengtong) [inline qemu/tests/pktgen.py]: Thanks for remind. updated.

**Comment by luckyh** [comment]
LGTM.

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by CongLi** [comment]
Thanks @Zhengtong @luckyh .

**Pull request merged**